images/installer/Dockerfile.upi.ci

```diff
@@ -45,10 +45,10 @@ ENV CLOUDSDK_PYTHON=/usr/bin/python
 
 RUN python3 -m pip install yq
 
-ENV TERRAFORM_VERSION=0.12.24
+ENV TERRAFORM_VERSION=1.0.11
 RUN curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
     unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin/
-ENV MATCHBOX_PROVIDER_VERSION=v0.3.0
+ENV MATCHBOX_PROVIDER_VERSION=v0.5.0
 RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/download/${MATCHBOX_PROVIDER_VERSION}/terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux-amd64.tar.gz && \
```

Comment:
We should change this to: https://github.com/poseidon/terraform-provider-matchbox/releases/download/v0.5.0/terraform-provider-matchbox_0.5.0_linux_amd64.zip and use unzip.